### PR TITLE
UI tweak for followed hashtags (incorporates all the other changes for following hashtags)

### PR DESCRIPTION
**Pull request by danschwarz**
If you're following a hashtag, there's no indication of that in toot tui. This pull request adds limited support; any hashtag that you are following is highlighted in the status detail in a different color (yellow) than the other highlighted hashtags (cyan)

This pull request is a superset of pull request #265 so please accept one or the other, but not both.

### CODE REVIEW

**Comment by danschwarz** [comment]
Screenshot with my followed hashtag #NYC highlighted in yellow.

image

**Comment by ihabunek** [comment]
Don't worry about conflicts here caused by merging #265. I can just cherry pick the relevant commits.

If I merge this, am I going to break #277 yet again? :)

**Comment by danschwarz** [comment]
This and the ANSI image changes should only conflict in timeline.py and
that will be trivial to resolve. I can do a test merge later to confirm.

File: toot/tui/utils.py [hunk 1]
```diff
@@ -51,11 +51,18 @@ def _gen():
     return list(_gen())
 
 
-def highlight_hashtags(line, attr="hashtag"):
-    return [
-        (attr, p) if p.startswith("#") else p
-        for p in re.split(HASHTAG_PATTERN, line)
-    ]
+def highlight_hashtags(line, followed_tags, attr="hashtag",\
```
Comment (ihabunek) [inline toot/tui/utils.py:54]: You don't need a backspace for content inside parenthesis. I mostly avoid using backslashes.

File: toot/tui/app.py [hunk 2]
```diff
@@ -331,6 +333,28 @@ def _done(instance):
 
         return self.run_in_thread(_load_instance, done_callback=_done)
 
+    def async_load_followed_tags(self):
+        def _load_tag_list():
+            logger.info("Loading tags")
```
Comment (ihabunek) [inline toot/tui/app.py:338]: I removed the logging. There's debug logging when accessing the API so it's redundant here.

**Review by ihabunek — COMMENTED** [review]
Thanks. I merged this manually.

BTW, I added [contribution guidelines](https://github.com/ihabunek/toot/blob/master/CONTRIBUTING.md) recently. Please check them out when you get a chance.

**Pull request closed without merging**